oci/go/image.yaml

```diff
@@ -0,0 +1,20 @@
+version: 2
+
+upload:
+  - source: canonical/golang-rock
+    directory: go-rock/1.22-24.04
+    commit: 8df1631ffae0eb023fca60379756311d7fd4b952
+    release:
+      1.22-24.04:
+        end-of-life: "2029-05-31T00:00:00Z"
+        risks:
+          - edge
```

Comment:
exceptionally, make it go to stable already, given the hard deadline and the lack of existing promotion processes